symbols/arm9.yml

```diff
@@ -2451,6 +2480,29 @@ arm9:
         
         r0: [output] bool array?
         r1: number of items to sort (usually just the current size of storage)
+    - name: AllKecleonShopsZInit
+      address:
+        NA: 0x2010594
+      description: |-
+        Empties the Kecleon shop for both TEAM_MAIN and TEAM_SPECIAL_EPISODE. TEAM_RESCUE does not appear to have its own
+        Kecleon shop.
+        
+        No params.
+    - name: SpecialEpisodeKecleonShopZInit
+      address:
+        NA: 0x2010654
+      description: |-
+        Empties the special episode Kecleon shop.
+        
+        No params.
+    - name: SetActiveKecleonShop
```

Comment:
Note: I am not sure if I need to clarify that it doesn't seem like there is a Kecleon Shop inventory for the Rescue Team and that the game will instead call SetActiveKecleonShop(TEAM_MAIN) when entering rescue mode. (I would assume there isn't a Kecleon Shop for the rescue team, but I technically have not checked if it does something bizarre like have the space for it in GAME_STATE_VALUES but just not use it as I don't know what comes directly after the special episode Kecleon shop).